create-table-games.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS games;
+
+CREATE TABLE games (
+  id                    serial,
+  matchup               text,
+  time_of_first_pitch   text,
+  experience_rating     text,
+  lowest_ticket_price   text,
+  average_ticket_price  text,
+  link_to_buy_tickets   text
+);
```

Comment:
reconsider the column types here, are any of these numeric?
https://www.postgresql.org/docs/9.5/static/datatype.html